templates/meta

```diff
@@ -0,0 +1,17 @@
+{
+    "perl" : "6.c",
+    "name" : "Nondescript",
+    "version" : "0.0.1",
+    "description" : "Nondescript.",
+    "authors" : [
+        "John Doe"
```

Comment:
Maybe put "Author Name" instead?

Reply:
I used "Jonh Doe" because that's the standard way to refer to an anonymous name, but I can change this.